base/applications/network/tracert/lang/it-IT.rc

```diff
@@ -1,26 +1,32 @@
+/*
+ * PROJECT:     ReactOS trace route utility
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     TRACERT translation file in Italian
+ * COPYRIGHT:   Copyright 2018 Bișoc George (fraizeraust99 at gmail dot com)
+ */
```

Comment:
I don't think we use SPDX headers for any `.rc` files.

Reply:
I'm not sure there are rules for that in our "coding style". I would personally like to see such headers at least to track who wrote the translations.

Reply:
@ColinFinck, any advice on that?

Reply:
Better to have more than less license headers! Ideally, every single file gets a license header. Otherwise, if a file was imported into another project, it would carry no information about its original author and license.

For instance, I know that "getbuilds" and "testman" translations use license headers in their translations too. Would be surprised if those were the only ones.